packages/plugins/rum/src/privacy/index.ts

```diff
@@ -49,6 +55,9 @@ export const getPrivacyPlugin = (
         buildEnd: () => {
             log.debug(`Instrumentation result - privacy dictionary size: ${dictionarySize}`, {
```

Comment:
This is a bit misleading; we are adding up the size of the privacy dictionary in each individual file and calling it the total privacy dictionary size, but in fact, duplicate entries in different files are deduplicated at runtime, so the actual privacy dictionary size will be smaller.

It might make sense to reword this to better clarify what's happening; perhaps something like:
```suggestion
            log.debug(`Privacy dictionary will include ${dictionaryEntryCount} entries across ${fileCount} files`, {
```